osprey/processes/wps_convert.py

```diff
@@ -47,8 +73,31 @@ def __init__(self):
             status_supported=True,
         )
 
+    def edit_config_file(self, config_file, uhs_files, station_file, domain):
+        parser = configparser.ConfigParser()
+        parser.optionxform = str
+
+        unprocessed = config_file
+        config_dict = read_config(unprocessed)
+        for section in config_dict.keys():
+            parser[section] = {
+                k: str(config_dict[section][k]) for k in config_dict[section].keys()
+            }
+
+        parser["UHS_FILES"]["ROUT_DIR"] = "/".join(uhs_files.split("/")[:-1])
+        parser["UHS_FILES"]["STATION_FILE"] = station_file
+        parser["DOMAIN"]["FILE_NAME"] = domain
+
+        processed = ".".join(unprocessed.split(".")[:-1]) + "_edited.cfg"
+        with open(processed, "w") as cfg:
+            parser.write(cfg)
+
+        return processed
+
     def _handler(self, request, response):
-        loglevel = request.inputs["loglevel"][0].data
+        args = collect_args(request, self.workdir)
+        loglevel, uhs_files, station_file, domain, config_file = tuple(args.values())
```

Comment:
While this does solve the comment I made above, we are straining things a bit harder than we should. Instead of turning the values from `args` into a tuple, could we not just make `args` a tuple in the first place?

Reply:
It is challenging because the size of `collect_args`'s return value differs depending on the optional inputs. Since I chose to take the advantage of dynamically defined args, I think this is the best idea. Perhaps you have any suggestions?

Reply:
That is a fair point. I think it is fine then, it may be helpful to condense the code into a single like if you can though. Otherwise I think it should be ok.